package.json

```diff
@@ -86,7 +86,7 @@
     "@fortawesome/react-fontawesome": "^0.1.3",
     "@portis/web3": "^2.0.0-beta.43",
     "@sentry/browser": "^5.0.8",
-    "@toruslabs/torus-embed": "^1.2.3",
+    "@toruslabs/torus-embed": "^1.2.6",
```

Comment:
should this be 1.3.0 ? (latest)

Reply:
that is coming in another PR.  No need here.

Reply:
in any case, 1.30 is badly broken for us.  we have to wait on it.